redisvl/index.py

```diff
@@ -317,6 +318,7 @@ def connect(self, url: Optional[str] = None, **kwargs):
             ValueError: If the REDIS_URL env var is not set and url is not provided.
         """
         self._redis_conn = get_redis_connection(url, **kwargs)
+        return self
```

Comment:
this feels odd given that then... it's a classmethod ... kindaof

Is this to make things more fluent?

Reply:
yes, it reduces the potential lines of code for a user by allowing `index = SearchIndex.from_dict(...).connect(...)`